wai-gems/wai-gems.gemspec

```diff
@@ -7,8 +7,7 @@ Gem::Specification.new do |spec|
   spec.email         = ["steve@opendirective.com"]
 
   spec.summary       = "Gem for dependencies of WAI theme gems."
-  # Use old version of rouge as latest jekyl defaults to 4.0.0 that requires ruby 2.7.0 and we use 2.6.2
-  spec.add_dependency "rouge", '3.30.0'
+  spec.add_dependency "rouge"
```

Comment:
@howard-e What about removing this line completely?

Reply:
@remibetin yep that could be done as well

Reply:
Removed it in https://github.com/howard-e/wai-website-data/pull/1/commits/d9e6e3b348640f8a6407c28a4393441c0d570b3a and updated https://github.com/howard-e/wai-website/pull/1 to confirm wai-website builds without issue.